tox.ini

```diff
@@ -16,6 +17,7 @@ deps=
   pytest-asyncio
 
 commands=
+  python setup.py develop
```

Comment:
tox uses `python2.7 setup.py sdist` which fails hard due `raise RuntimeError("aio-pika doesn't support Python version prior 3.4")`. So we need add pack python setup.py setup to get requirements installed inside tox. If `skipsdist = True` `extras` is not triggered.